test-suite/src/dictionary_index.rs

```diff
@@ -0,0 +1,22 @@
+use {crate::*, gluesql_core::prelude::Value::*};
+
+test_case!(ditionary_index, async move {
+    run!("CREATE TABLE Foo (id INT);");
+    run!("CREATE INDEX Foo_id ON Foo (id ASC)");
+    run!("CREATE INDEX Foo_id_1 ON Foo (id + 1 DESC)");
+    run!("CREATE INDEX Foo_id_2 ON Foo (id * 2)");
+
+    let test_cases = [(
+        "SELECT * FROM GLUE_INDEXES",
+        Ok(select!(
+            TABLE_NAME       | INDEX_NAME           | ORDER             | EXPRESSION;
+            Str              | Str                  | Str               | Str;
+            "Foo".to_owned()   "Foo_id".to_owned()    "BOTH".to_owned()    "id".to_owned();
+            "Foo".to_owned()   "Foo_id_1".to_owned()  "BOTH".to_owned()   "id + 1".to_owned();
```

Comment:
Looks like ASC, DESC don't work.. 🤔

Reply:
`SELECT * FROM GLUE_INDEXES ORDER BY INDEX_NAME DESC`
I see the test fails when I run it with , so it seems to work.

```shell
thread 'dictionary_index' panicked at 'assertion failed: `(left == right)`
   left: `Str("Foo_id_2")`,
  right: `Str("Foo_id")`:
[err: value] row index: 0
expected: [Str("Foo"), Str("Foo_id"), Str("BOTH"), Str("id")]
    found: [Str("Foo"), Str("Foo_id_2"), Str("BOTH"), Str("id * 2")]', test-suite/src/tester/mod.rs:81:17
```

Reply:
I meant the DESC which is the value of ORDER (BOTH, ASC, DESC) not the keyword of SQL 😢
Whenever we create index, the type of order is stored in `SchemaIndex`